src/lib/parts.ts

```diff
@@ -199,7 +207,7 @@ export class AttributeCommitter {
         // Native Symbols throw if they're coerced to string.
         value = String(value);
       }
-      this.element.setAttribute(this.name, value as string);
+      this.element.setAttributeNS(this.namespace, this.name, value as string);
```

Comment:
Let's be on the lookout for a perf regression here since `setAttributeNS` may not be as optimized as `setAttribute`.